pkg/kubectl/resource_printer.go

```diff
@@ -342,25 +342,41 @@ type HumanReadablePrinter struct {
 }
 
 // NewHumanReadablePrinter creates a HumanReadablePrinter.
-func NewHumanReadablePrinter(noHeaders, withNamespace bool, wide bool, showAll bool, showLabels bool, absoluteTimestamps bool, columnLabels []string) *HumanReadablePrinter {
+func NewHumanReadablePrinter(printOptions *PrintOptions) *HumanReadablePrinter {
+	if printOptions == nil {
+		printOptions = &PrintOptions{}
+	}
 	printer := &HumanReadablePrinter{
 		handlerMap: make(map[reflect.Type]*handlerEntry),
-		Options: PrintOptions{
-			NoHeaders:          noHeaders,
-			WithNamespace:      withNamespace,
-			WithKind:           false,
-			KindName:           "",
-			Wide:               wide,
-			ShowAll:            showAll,
-			ShowLabels:         showLabels,
-			AbsoluteTimestamps: absoluteTimestamps,
-			ColumnLabels:       columnLabels,
-		},
+		options:    *printOptions,
```

Comment:
> Looks like the problem is still there. Did you fix your code based on what those hack/verify-*.sh told you to do and re-run them to make sure everything passes?

Yes, `make verify` also passes locally, I will rebase and try again.

Also, part of the changes include making options a non-exported field again, as it was only exported in a previous PR in order to set a resource's `Kind` value from another package. I realize this might not have been the cleanest way of doing it, so this update is aiming to fix this:

https://github.com/kubernetes/kubernetes/pull/28509/files#diff-0fc00b94def20930bb77075c7e2c7599R340